var/spack/repos/builtin/packages/py-llvmlite/package.py

```diff
@@ -24,6 +25,7 @@ class PyLlvmlite(PythonPackage):
     version('0.20.0', sha256='b2f174848df16bb9195a07fec102110a06d018da736bd9b3570a54d44c797c29')
 
     depends_on('py-setuptools', type='build')
+    depends_on('python@3.7:3.10', type=('build', 'run'), when='@0.38.0:')
     depends_on('python@3.7:3.9', type=('build', 'run'), when='@0.37.0:')
```

Comment:
```suggestion
    depends_on('python@3.7:3.9', type=('build', 'run'), when='@0.37')
```
Otherwise this condition will also be applied to 0.38+

Reply:
Thanks

Reply:
Still waiting for this change

Reply:
The rebase must have undone that again. Sorry for that.